### ISSUE TITLE
Weekly digest - 4th Week'Feb 21

### ISSUE THREAD

**Issue by jdpsen** [primeqa/primeqa#140]
_jaydesen created the following on Feb 28:_  

 
 

_Original issue: https://github.ibm.com/ai-foundation/PrimeQA/issues/48_

**Comment by jdpsen** [comment]
_On Feb 28, jaydesen commented:_
Table+text QA (Vishwajeet)
 	- experimenting with GENRE based entity linking for row->passage linking. 
 	- issues with GENRE implementation, investigation in progress.

**Comment by jdpsen** [comment]
_On Feb 28, jaydesen commented:_
QG (saneem)
 	- using the parallel data from HybridQA-OTTQA for training "decontextualizer" module to produce decontextualized questions after T5 based QG.
 	- experimentation in progress
 	- discussed possible steps for Table+text based QG

**Comment by jdpsen** [comment]
_On Feb 28, jaydesen commented:_

Multi-lingual Retriver
	- [Martin, Yulong] best numbers obtained on XOR-Retrieve - gone to NAACL paper
	- brainstorming on what we should try next 
		options: 
			(1) Benchmarking the number on Task3 of XOR-tydi i.e. full multilingual QA with retriever + reader 
			(2) should we submit our system in this NAACL workshop ? https://mia-workshop.github.io/shared_task.html
	- [Bhavani] best numbers in Mr Tydi
		- 	ColBERT xlmr-nq model +5 on MRR and +6 on RECALL compared to the best baseline system. 
		- Fine-tuning on MrTyDI additional +11 for MRR and +0.4 for Recall

**Comment by jdpsen** [comment]
_On Feb 28, jaydesen commented:_
One-QA framework (Anthony)
	- stub packages created for disjoint set of functionalities/models to be inducted to One-QA framework for V1 (ACL deadline)
	- MRC pipeline implementation in progress

**Comment by jdpsen** [comment]
_On Feb 28, jaydesen commented:_
Generation Based QA
	-  [Mihela, Arafat] experiments in progress for using retriever+reranker for better answer generation. Retriever+generator does not yield good score. 
	- [Sarah] "claim annotation" task in progress to explore a new metric for evaluating answer generation based on " # of validated claim"
	-  [Scott] discussed about "Answer Diversity" metric inspired by Strategy QA

**Comment by jdpsen** [comment]
_On Feb 28, jaydesen commented:_

ListQA 
	- [IBM internal annotators] Pilot annotation task (of 20 questions) assigned to internal annotators to ensure high quality test sets for list question answering using 
	  GooAQ, CCQA, and NQ
	- [Sarah] Data cleanup for GooAQ, CCQA, NQ to produce an accurate set of questions/models for listQA
